webrender/src/frame.rs

```diff
@@ -319,6 +319,32 @@ impl Frame {
                 continue;
             }
 
+           let mut delta:Point2D<f32> = match scroll_location {
+            ScrollLocation::Delta(delta) => delta,
+            ScrollLocation::Start => {
+                if layer.scrolling.offset.y.round() == 0.0 {
+                    // Nothing to do.
+                    return false;
```

Comment:
I *think* that we don't want to return straight away here (or below) due to the recent scroll root changes that have been made. Is that correct @mrobinson  ?